assets/scss/components/_ie-fixes.scss

```diff
@@ -0,0 +1,95 @@
+html.ie9 {
+    .Header {
+        height: 50px;
+
+        > * {
+            float: left;
+        }
+    }
+
+    .Frame-handle {
+        display: none;
+    }
+
+    .Frame-body {
+        height: auto;
+        width: 100%;
+    }
+    .Frame-inner {
+        margin-top: 50px;
+    }
+    .Frame-panel {
+        overflow: auto;
+    }
+    .Frame-panel--sidebar {
```

Comment:
I choose it just to make if as fast as possible, adding fixed elements is possible, but I'll need more time to do it